BlueprintUI/Sources/Internal/LayoutModeKey.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 enum LayoutModeKey: EnvironmentKey {
-    static let defaultValue: LayoutMode = .default
+    static var defaultValue: LayoutMode { .default }
```

Comment:
Out of curiosity, why was this change made?

Reply:
> Out of curiosity, why was this change made?

`LayoutMode.default` is now a `var`, so we've gotta reference that dynamically rather than copying the initial value.